lang/python.md

```diff
@@ -2,10 +2,157 @@
 lang: Python
 syntax: python
 title: Python is a snake language!
+author : AaquibAli
 ---
 
 ```python
 print("hello world")
 ```
+ ## What is Python?
```

Comment:
There's an extra space here which will make the header not work well.
```suggestion
## What is Python?
```

Reply:
@AaquibAli: As Pan requested, you've to change this as well!
I forgot this changes to let you made in voice chat! XD
My bad!